src/main/java/org/folio/rest/client/FeeFinePubSubClient.java

```diff
@@ -0,0 +1,71 @@
+package org.folio.rest.client;
+
+import static io.vertx.ext.web.client.WebClient.create;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.HttpStatus;
+import org.folio.rest.exception.InternalServerError;
+import org.folio.rest.jaxrs.model.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpResponse;
+
+public class FeeFinePubSubClient {
+  private static final Logger log = LoggerFactory.getLogger(FeeFinePubSubClient.class);
+  private final OkapiClient okapiClient;
+
+  public FeeFinePubSubClient(Vertx vertx, Map<String, String> okapiHeaders) {
```

Comment:
Does the prebuilt pub-sub client not do this already, or is it inappropriate because it does not give enough control over interpreting the response?

Reply:
The prebuilt pub/sub client does not ignore the 400 `No subscribers for the event type` error. But this overridden client ignores it.

Reply:
Thanks @bohdan-suprun